p2p/transport/quicreuse/reuse.go

```diff
@@ -97,15 +143,15 @@ func newReuse() *reuse {
 func (r *reuse) gc() {
 	defer func() {
 		r.mutex.Lock()
-		for _, conn := range r.globalListeners {
-			conn.Close()
+		for _, tr := range r.globalListeners {
+			tr.Close()
 		}
-		for _, conn := range r.globalDialers {
-			conn.Close()
+		for _, tr := range r.globalDialers {
+			tr.Close()
 		}
-		for _, conns := range r.unicast {
-			for _, conn := range conns {
-				conn.Close()
+		for _, trs := range r.unicast {
+			for _, tr := range trs {
+				tr.Close()
 			}
```

Comment:
Not sure how to comment on unchanged code. Can you fix naming on Line 170. That's still using Conns.